### PR TITLE
Added classic_link and classic_link_dns options

### PR DESCRIPTION
**Pull request by orimarti**
Added classic_link and classic_link_dns options

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -55,6 +55,16 @@ variable "enable_dns_support" {
   default     = false
 }
 
+variable "enable_classiclink" {
+  description = "should be true if you want to use ClassicLink within the VPC"
+  default     = true
```
Comment (antonbabenko) [inline variables.tf]: Keep this default to `false` to be backward compatible, also Terraform uses `false` as default.

**Review by antonbabenko — CHANGES_REQUESTED** [review]

**Comment by antonbabenko** [comment]
By any chance, can you use the [terraform-aws-vpc](https://github.com/terraform-aws-modules/terraform-aws-vpc/) which is much more up-to-date and feature-rich.

PS: This repo is not maintained any longer.

File: variables.tf (hunk 1 above)
Comment (orimarti) [reply variables.tf]: Changed.

**Comment by orimarti** [comment]
Yes, I know this is deprecated what we are using this one right now and think there's not an easy way to migrate to the new one without breaking anything no?

**Comment by antonbabenko** [comment]
There is a fairly easy way to migrate from this repo to the new one. After you change the source and run `terraform plan` with the new source and the same (or slightly changed arguments to the module) you will see that some resources should be recreated because names have been changed for them.

You will have to run few `terraform state mv` commands as described [in this issue](https://github.com/terraform-aws-modules/terraform-aws-vpc/issues/5) to get state updated, so that terraform plan see no difference.

**Review by antonbabenko — APPROVED** [review]

**Comment by antonbabenko** [comment]
PS: I've just released [v1.0.14](https://github.com/terraform-community-modules/tf_aws_vpc/releases/v1.0.14) of this module.

**Pull request merged**